eclass/qmake-utils.eclass

```diff
@@ -16,16 +17,16 @@
 if [[ -z ${_QMAKE_UTILS_ECLASS} ]]; then
 _QMAKE_UTILS_ECLASS=1
 
-[[ ${EAPI:-0} == [012345] ]] && inherit eutils multilib
+[[ ${EAPI:-0} == [01234] ]] && die "qmake-utils.eclass: unsupported EAPI=${EAPI:-0}"
```

Comment:
did you check all consumers? including those in gentoo.git?

Reply:
Yes, there was only one left which I removed. https://qa-reports.gentoo.org/output/eapi-per-eclass/qmake-utils.eclass/

Reply:
kde-sunset.git already contains copies of Qt4 ebuilds and qt4*eclass, current state of qmake-utils.eclass in tree should also be copied there before syncing overlay changes.

Reply:
(qt.git and kde.git have already banned <6, so I did not have to check)